docs/translations.rst

```diff
@@ -3,12 +3,33 @@ Internationalization
 
 This page contains the most common techniques needed for managing CommCare HQ
 localization strings. For more comprehensive information, consult the
-`Django Docs translations page <https://docs.djangoproject.com/en/dev/topics/i18n/translation/>`_.
+`Django Docs translations page <https://docs.djangoproject.com/en/dev/topics/i18n/translation/>`_
+or `this helpful blog post <http://blog.bessas.me/post/65775299341/using-gettext-in-django>`_.
 
 
 Tagging strings in views
 ------------------------
 
+**TL;DR**: ``ugettext`` should be used in code that will be run per-request.
+``ugettext_lazy`` should be used in top-level code.
```

Comment:
The meaning of "top-level code" isn't very clear, although I can infer from the context that it is code that is run at module import time or generally outside of a request. Aside: do celery tasks have a locale?